opacclient/gradle.properties

```diff
@@ -11,9 +11,10 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.jvmargs=-Xmx2048m
+#org.gradle.jvmargs=-Xmx2048m
 
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+org.gradle.jvmargs=-XX\:MaxHeapSize\=256m -Xmx256m
```

Comment:
Could you try to put your `org.gradle.jvmargs=-XX\:MaxHeapSize\=256m -Xmx256m` setting in a `gradle.properties` file in your home directory ("/home/username" on Linux, "C:\Users\username" on Windows), leaving the `gradle.properties` in the project directory as it was before? The file from your home directory should be able to override the settings.